packages/gatsby/cache-dir/navigation.js

```diff
@@ -172,6 +172,8 @@ class RouteAnnouncer extends React.Component {
         pageName = document.title
       }
       const pageHeadings = document
+        .getElementById(`gatsby-focus-wrapper`) &&
+        document
         .getElementById(`gatsby-focus-wrapper`)
         .getElementsByTagName(`h1`)
```

Comment:
Maybe we can simply do this:

```suggestion
      const pageHeadings = document
        .querySelectorAll(`#gatsby-focus-wrapper h1`)
```

@madalynrose What's your opinion on this?

Reply:
fwiw, looks better to me than what I had.

Reply:
yeah that looks good to me. nice solution @vladar